packages/manager/apps/public-cloud/src/index.js

```diff
@@ -77,7 +77,9 @@ angular
   })
   .run(/* @ngInject */ ($state) => {
     $state.defaultErrorHandler((error) => {
-      if (error.type === RejectType.ERROR) {
+      console.log('error', error);
```

Comment:
?

Reply:
just added to check something in staging, its removed now

Reply:
as of now readonly user can only be tested in staging